salt/_roster/kubernetes_nodes.py

```diff
@@ -0,0 +1,61 @@
+import fnmatch
+import logging
+
+try:
+    import kubernetes.client
+    import kubernetes.config
+    HAS_DEPS = True
+except ImportError:
+    HAS_DEPS = False
+
+
+log = logging.getLogger(__name__)
+
+
+__virtualname__ = 'kubernetes'
+
+
+def __virtual__():
+    if HAS_DEPS:
+        return HAS_DEPS
+    else:
+        return False, 'Unable to load the kubernetes nodes roster: '\
+            'dependencies kubernetes.client and/or '\
```

Comment:
minor: "Kubernetes Python client library" may have been clearer than "kubernetes.client and/or kubernetes.config"